controller/loginController.js

```diff
@@ -0,0 +1,71 @@
+const User = require('../models/user')
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt')
+const http = require('http-status-codes')
+const secretKey = require('../config/secretKey')
+const statusMsg = require('../config/statusMsg')
+const express = require('express')
+const app = express()
+
+exports.login = async (req, res) => {
+    let user = await User.findOne({ email: req.body.email })
+    if (!user) res.status(http.BAD_REQUEST).json({ message: http.getStatusText(http.BAD_REQUEST) })
+    var result = await bcrypt.compare(req.body.password, user.password)
```

Comment:
use let instead of var. try to use es6 more.

Reply:
done